ui/components/multichain/network-list-menu/network-list-menu.js

```diff
@@ -227,7 +260,93 @@ export const NetworkListMenu = ({ onClose }) => {
                 {t('noNetworksFound')}
               </Text>
             ) : (
-              generateMenuItems(searchResults)
+              <DragDropContext onDragEnd={onDragEnd}>
+                <Droppable droppableId="characters">
+                  {(provided) => (
+                    <Box
+                      className="characters"
+                      {...provided.droppableProps}
+                      ref={provided.innerRef}
+                    >
+                      {searchResults?.map((network, index) => {
+                        if (
```

Comment:
Note to self -- we should see if this is still necesssary.

Reply:
removed it here https://github.com/MetaMask/metamask-extension/pull/21163/commits/cd4049f31e56fa9cf0360157f96511629070e83e